pkgs/development/libraries/directfb/default.nix

```diff
@@ -25,6 +25,14 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  postPatch = ''
+    # https://github.com/deniskropp/DirectFB/blob/master/src/core/Makefile.am#L15
+    # BUILDTIME is embedded in the result
+    # if switching to cmake then a similar substitution has to be done
+    substituteInPlace src/core/Makefile.am \
+      --replace '`date -u "+%Y-%m-%d %H:%M"`' "`date -u \"+%Y-%m-%d %H:%M\" --date="@''${SOURCE_DATE_EPOCH}"`"
```

Comment:
Is it intentional to let the date expand at substitution time as opposed to "make" time (2nd parameter to --replace is in double quotes instead of single quotes)?

Reply:
Yes, but it also doesn't matter